ocaml/quicktest/quicktest_vm_import_export.ml

```diff
@@ -81,7 +70,7 @@ let import_export_test rpc session_id sr_info vm_template () =
                 if all.API.vBD_empty <> orig_vbd.API.vBD_empty
                 then Alcotest.fail (Printf.sprintf "Device %s varies in emptiness" all.API.vBD_userdevice);
                 match all.API.vBD_userdevice with
```

Comment:
Should this not use a wildcard to catch unexpected value? Otherwise we would get a match runtime exception.

Reply:
There is one below.